### DISCUSSION TITLE
Module parse failed

### DISCUSSION THREAD

**Discussion by AntonMarkhotski** [DoneDeal0/Talkr#31] [Q&A] [answered]
Hi!
I,m trying to start my project with installed talkr package. And I have this problem

Compiled with problems:X

ERROR in ./node_modules/talkr/dist/index.es.js 1:0

Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (1:0)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
 * ./node_modules/source-map-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
> import * as e from "react";
| import { createContext as n, useState as t, useContext as r } from "react";
| var a = function () {

**Comment by DoneDeal0** [comment, marked as answer]
Hi @AntonMarkhotski !

Thanks for your interest in Talkr! Do you use Create React App? I didn't have any issue with the new build on a codesandbox or on custom React projects, so I assumed everything was fine. But I rollbacked to a previous build version. It should work now. I've just tested it with CRA, everything looks fine.

Please make sure you have installed version `3.3.9`. If the issue persists, please let me know. You can use `3.3.7` meanwhile (write `talkr: "3.3.7"` and not `"talkr": "^3.3.7"` in your `package.json` to force the version `3.3.7`). But `3.3.9` should be fine.

**Reply by AntonMarkhotski** [reply]
Hi! Thanks you! Now it's work fine!
